UnityProject/Assets/Scripts/Core/SecureStuff/SecureMapsSaver.cs

```diff
@@ -269,5 +332,70 @@ public static class SecureMapsSaver
 				Loggy.LogError(e.ToString());
 			}
 		}
+
+		private static List<object> IEnumeratorToList(IEnumerator IEnumerable)
+		{
+			var List = new List<object>();
+			try
+			{
+				bool loop = true;
```

Comment:
You're not using this. Also, an exit strategy would be nice in case the loop gets stuck.

Reply:
Alrighty